package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "hof",
   "description": "A bootstrap for HOF projects",
-  "version": "20.1.17",
```

Comment:
Can you bump this to a minor version.  We use semantic versioning and I'd say this is adding functionality rather than a patch.

Even if it didn't we try to now do minor updates and have our services use tilde instead of carets so minor updates won't effect us going forward unless we update our versioning